examples/hello/bpReader/helloBPReader.cpp

```diff
@@ -64,12 +74,15 @@ int main(int argc, char *argv[])
             bpIO.InquireVariable<float>("bpFloats");
         adios2::Variable<int> bpInts = bpIO.InquireVariable<int>("bpInts");
 
+        unsigned long  Nx = 10;
```

Comment:
I've got no issue with moving the declaration of `Nx` to a more localized scope like this, it's a generally a good practice, so please keep that.  However, the type should remain `const std::size_t`

Reply:
@chuckatkins : Yup, this was an oversight on my part. Fixed.